api/clients/v2/verification/cert_verifier.go

```diff
@@ -104,3 +109,26 @@ func (cv *CertVerifier) VerifyCertV2(
 
 	return nil
 }
+
+// GetNonSignerStakesAndSignature calls the getNonSignerStakesAndSignature view function on the EigenDACertVerifier
+// contract, and returns the resulting NonSignerStakesAndSignature object.
+func (cv *CertVerifier) GetNonSignerStakesAndSignature(
+	ctx context.Context,
+	signedBatch *disperser.SignedBatch,
+) (*verifierBindings.NonSignerStakesAndSignature, error) {
+
+	signedBatchBinding, err := SignedBatchProtoToBinding(signedBatch)
+	if err != nil {
+		return nil, fmt.Errorf("convert signed batch: %s", err)
+	}
```

Comment:
shouldn't we be using `%w` instead of `%s` for error -> string formatting?

Reply:
Yes, thanks for catching [c08d16f8](https://github.com/Layr-Labs/eigenda/pull/1159/commits/c08d16f8e7b62275763a018dd9fc153284303f86)